File: app/src/main/java/com/commit451/easel/sample/MainActivity.java
package com.commit451.easel.sample;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.SwitchCompat;
import android.support.v7.widget.Toolbar;
import android.view.View;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.EditText;
import android.widget.ProgressBar;
import android.widget.RadioButton;
import android.widget.ScrollView;
import android.widget.SeekBar;

import com.commit451.easel.Easel;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        toolbar.setTitle("Easel");
        toolbar.inflateMenu(R.menu.menu_main);
        Easel.tint(toolbar.getMenu(), Color.MAGENTA);
        Easel.tintOverflow(toolbar, Color.MAGENTA);

        ScrollView scrollView = (ScrollView) findViewById(R.id.scrollview);
        Easel.tintEdgeEffect(scrollView, Color.MAGENTA);

        EditText editText = (EditText) findViewById(R.id.editText);
        RadioButton radioButton = (RadioButton) findViewById(R.id.radioButton);
        CheckBox checkBox = (CheckBox) findViewById(R.id.checkBox);
        SwitchCompat switchCompat = (SwitchCompat) findViewById(R.id.switchCompat);
        ProgressBar progressBar = (ProgressBar) findViewById(R.id.progressBar);
        SeekBar seekBar = (SeekBar) findViewById(R.id.seekBar);
        Button button = (Button) findViewById(R.id.button);
        Easel.tint(editText, Color.MAGENTA);
        Easel.tint(radioButton, Color.MAGENTA);
        Easel.tint(checkBox, Color.MAGENTA);
        Easel.tint(switchCompat, Color.MAGENTA);
        Easel.tint(progressBar, Color.MAGENTA);
        Easel.tint(seekBar, Color.MAGENTA);
        Easel.tint(button, Color.MAGENTA);
        findViewById(R.id.button_kotlin).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, KotlinActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/commit451/easel/sample/MainActivity.kt
package com.commit451.easel.sample

import android.graphics.Color
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.SearchView
import androidx.appcompat.widget.SwitchCompat
import androidx.appcompat.widget.Toolbar
import android.view.View
import android.widget.*
import com.commit451.easel.tint
import com.commit451.easel.tintEdgeEffect
import com.commit451.easel.tintOverflow


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val toolbar = findViewById<View>(R.id.toolbar) as Toolbar
        toolbar.title = "Easel"
        toolbar.inflateMenu(R.menu.menu_main)
        val searchMenuItem = toolbar.menu.findItem(R.id.action_search)
        val searchView = searchMenuItem.actionView as SearchView

        toolbar.menu.tint(Color.MAGENTA)
        toolbar.tintOverflow(Color.MAGENTA)

        val scrollView = findViewById<View>(R.id.scrollview) as ScrollView
        scrollView.tintEdgeEffect(Color.MAGENTA)

        val editText = findViewById<View>(R.id.editText) as EditText
        val radioButton = findViewById<View>(R.id.radioButton) as RadioButton
        val checkBox = findViewById<View>(R.id.checkBox) as CheckBox
        val switchCompat = findViewById<View>(R.id.switchCompat) as SwitchCompat
        val progressBar = findViewById<View>(R.id.progressBar) as ProgressBar
        val seekBar = findViewById<View>(R.id.seekBar) as SeekBar
        val button = findViewById<View>(R.id.button) as Button
        val spinner = findViewById<View>(R.id.spinner) as Spinner
        editText.tint(Color.MAGENTA)
        radioButton.tint(Color.MAGENTA)
        checkBox.tint(Color.MAGENTA)
        switchCompat.tint(Color.MAGENTA)
        progressBar.tint(Color.MAGENTA)
        seekBar.tint(Color.MAGENTA)
        button.tint(Color.MAGENTA)
        spinner.tint(Color.MAGENTA)
    }
}
